components/TxForm.tsx

```diff
@@ -207,7 +208,7 @@ export const TxForm = ({ recipientAddress }: { recipientAddress?: string }) => {
             size="lg"
             mt=".75rem"
             width="100%"
-            isDisabled={chain?.unsupported}
+            isDisabled={!isConnected && chain?.unsupported}
```

Comment:
I think we want an `or` here? So the button is disable if not connected, or if the chain is unsupported
```suggestion
            isDisabled={!isConnected || chain?.unsupported}
```

Reply:
Actually I guess both end up behaving the same, since if not connected, the chain is unsupported?

Reply:
yea, that's what I thought, I think your correction is more correct though